recipes/boost/all/conandata.yml

```diff
@@ -240,5 +246,33 @@ patches:
     - patch_file: "patches/1.79.0-geometry_no_rtti.patch"
       base_path: "source_subfolder"
       patch_type: "portability"
-      url: "https://github.com/boostorg/geometry/commit/b354162798749b3aaa539755e7b9be49d2b9a9c4.patch"
       patch_source: "https://github.com/boostorg/geometry/discussions/1041"
+  1.80.0:
+    - patch_file: "patches/1.80.0-locale-fail-on-missing-backend.patch"
+      patch_description: "Fails the build when there is no iconv backend"
+      patch_type: "conan"
+      base_path: "source_subfolder"
+    - patch_file: "patches/boost_1_77_mpi_check.patch"
+      patch_description: "Fails the build when mpi is not configured"
+      patch_type: "conan"
+      base_path: "source_subfolder"
+    - patch_file: "patches/1.80.0-0001-filesystem-win-fix-dir-it-net-share.patch"
+      patch_description: "Directory iterators may fail to construct for a network share on Windows prior to 10"
+      patch_type: "official"
+      patch_source: "https://github.com/boostorg/filesystem/issues/245"
```

Comment:
I absolutely appreciate that someone took the time to add these link ❤️

Reply:
Thanks! They are super helpful for context and keeping track of what needs to be pulled forward.